docs/website/docs/dlt-ecosystem/verified-sources/freshdesk.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Freshdesk
+description: dlt verified source for Freshdesk API
+keywords: [freshdesk api, freshdesk verified source, freshdesk]
+---
+import Header from './_source-info-header.md';
+
+# Freshdesk
+
```

Comment:
you forgot about this
![image](https://github.com/dlt-hub/dlt/assets/20367975/11cb2015-2d0e-4675-bd1f-c0cf428dc460)

Reply:
just add
```
 <Header/>
```

Reply:
added